testing/cucumber/features/linters/test_with_no_verification_step.feature

```diff
@@ -24,8 +24,35 @@ Feature: Test with no verification step linter
       | linter                           | problem                           | location         |
       | TestWithNoVerificationStepLinter | Test does not have a 'Then' step. | <path_to_file>:3 |
 
+  Scenario: Configuration of keywords for different dialect
+    Given a linter for tests with no verification step has been registered
+    And the following configuration file:
+      """
+      TestWithNoVerificationStepLinter:
+        Given:
+          - Dado
+        When:
+          - Quando
+        Then:
+          - Então
+      """
```

Comment:
No need to add extra keywords one these ones because the linter is checking for the absence of something and so additional words don't do much. All good here.